admin-dev/themes/new-theme/js/components/modal/iframe-modal.ts

```diff
@@ -237,7 +237,7 @@ export class IframeModal extends Modal implements IframeModalType {
   }
 
   render(content: string, hideIframe: boolean = true): this {
-    this.modal.message.innerHTML = content;
+    this.modal.message.innerText = content;
```

Comment:
Except the purpose of this render method is also to render HTML, it extends the base method from the `ModalType` and has a few use cases already where it renders HTML code:
https://github.com/PrestaShop/PrestaShop/blob/3a0ae040c5642364f0d1535bd5769dcbdd31f8d5/admin-dev/themes/new-theme/js/pages/product/combination/combination-modal/CombinationModal.vue#L270
https://github.com/PrestaShop/PrestaShop/blob/9541a5198c41a2723abec7d431525e8d030d83d2/admin-dev/themes/new-theme/js/pages/product/category/category-tree-selector.ts#L91

Even if those examples are concrete implementation of a `Modal` and not an `IframeModal` they still share the same base type and so must respect the same contract/behaviour